res/values-fr/strings.xml

```diff
@@ -41,7 +41,7 @@
   <string name="prefs_remember_last_share_location">Mémoriser l\'emplacement de partage</string>
   <string name="prefs_remember_last_upload_location_summary">Mémoriser le dernier emplacement d\'upload</string>
   <string name="recommend_subject">Essayez %1$s sur votre smartphone&amp;nbsp;!</string>
-  <string name="recommend_text">J'aimerais vous inviter à utiliser %1$s sur votre smartphone !
+  <string name="recommend_text">J\'aimerais vous inviter à utiliser %1$s sur votre smartphone !
```

Comment:
Language translations should not be changed here directly, if not through Transifex. But maybe I think you needed to do that because there were some errors on this french tokens and the project did not build.
When you do the rebase on develop, this changes should disappear.